06_puppy/patrickmarabeas/syncStore_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSyncStore(t *testing.T) {
+
+	t.Run("NewSyncStore", func(t *testing.T) {
```

Comment:
Seems like these should just be separate top-level `Test…` functions.

Reply:
Tests have been refactored.